content/docs/user-guide/how-to/share-a-dvc-cache.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'How to Share a Cache Among Projects'
+description: >-
+  Setup a single cache for different projects or distributed copies of a same
+  project.
+---
+
+# How to Share a DVC Cache
+
+There are 2 main reasons to setup a shared <abbr>DVC cache</abbr>:
+
+1. You have distributed copies of a DVC repository in a single shared server
+   with multiple users. A shared cache is necessary to avoid duplicating the
+   project's data on the single local storage available to all.
+2. Your team works with multiple projects in environments with limited storage,
+   which share a large storage unit. Everyone needs to use the shared drive
+   anyway, and combining the cache locations will also prevent data duplication
+   (across projects).
+
+## Preparation
```

Comment:
Are there any changes to the things below, compared to the existing doc?

Reply:
Nope. The Example is gone though, as it doesn't make sense in a how-to.